### DISCUSSION TITLE
video stop function plz

### DISCUSSION THREAD

**Discussion by wonder21c** [gopro/labs#979] [Feature Requests]
**There is a function to start multiple cameras simultaneously, but how can I make it so that I can stop them at a desired time instead of setting a specific stop time in advance?**

**Comment by dnewman-gpsw** [comment]
Please explain the difference between "desired time" and "specific stop time".  Usage example might be helpful.

**Reply by wonder21c** [reply]
Currently, in the QR system, both start time and end time must be set. However, I only want to set the start time and leave the end time unspecified.

**Reply by dnewman-gpsw** [reply]
If you only want to set a start, do that.  Usage example might be helpful.

!S <- is the start command, there is no implied end.

**Reply by wonder21c** [reply]
I operate multiple cameras simultaneously. However, I don't know the exact end time. I would like to stop all the cameras at the same time as soon as filming is finished. Is there any way to do this other than pressing a button manually?

**Reply by dnewman-gpsw** [reply]
What would be the mechanism? You can make a script that is looking for the BLDE remote (I think support five cameras), turn the remote off to end capture. I've not tried that.  If you are ruling out time, you have to have another signal, like sound, motion, BLE etc.

**Reply by wonder21c** [reply]
Can the "The Remote" feature of GoPro, which allows control of up to 5 cameras, be extended to control more cameras using QR codes or another coding method?

**Reply by dnewman-gpsw** [reply]
No. If you tell me what you are planning, I can help better.

**Reply by wonder21c** [reply]
I am planning to shoot videos with more than 10 cameras. Turning them on and off one by one is very cumbersome, so I need to start and stop recording simultaneously. However, since I don't set a predetermined end time, I'm experiencing difficulties.

**Reply by dnewman-gpsw** [reply]
Options:

1) Wired USB.  Use a USB power trigger, have the cameras auto-record when USB power is on.  You can have a master switch to control all the cameras.

2) Do you need audio capture on the cameras? If you added MediaMod to each camera with audio line input, you can have a wire audio trigger to start and stop all the cameras at the same time. 

3) If you are shooting in a controlled environment, you have turn the lights on and off as a trigger to start and stop capture.

4) If you have a software developer on your team, you could use Open GoPro to add all the camera into a Wi-Fi network, and start and stop them programmatically. see: https://gopro.github.io/OpenGoPro/

5) If the cameras are pointing in approximately the same direction, use a large QR Code.  ```!E``` the end command will work if the cameras have been enabled to read QR codes during capture (not their default.)  To enable use *QRDR=1.

**Reply by wonder21c** [reply]
thank you It was very helpful. :)
